src/aks-preview/HISTORY.rst

```diff
@@ -11,6 +11,9 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 +++++++
+14.0.0b5
++++++++
+* Add aks extension and aks extension-type command groups
 
 14.0.0b6
 +++++++
```

Comment:
Please choose a version number (b5 and b6 have been released)

Reply:
changed to 14.0.0b7